tests/tests.js

```diff
@@ -1178,3 +1212,38 @@ function muxedTextTests(manifestUrl, format) {
     ]);
   });
 }
+
+function multiPeriodTests(manifestUrl, format) {
+  it('can process multiperiod_inputs_list ' + format, async() => {
+    const singleInputConfigDict = {
+      'inputs': [
+        {
+          'name': TEST_DIR + 'Sintel.with.subs.mkv',
+          'media_type': 'video',
+          // Keep this test short by only encoding 1s of content.
+          'end_time': '0:01',
+        },
+      ],
+    };
+    const inputConfigDict = {
+      'multiperiod_inputs_list': [
+        singleInputConfigDict,
+        singleInputConfigDict,
+      ],
+    };
+    const pipelineConfigDict = {
+      'streaming_mode': 'vod',
+      'resolutions': ['144p'],
+      'audio_codecs': ['aac'],
+      'video_codecs': ['h264'],
+    };
+
+    await startStreamer(inputConfigDict, pipelineConfigDict);
+    await player.load(manifestUrl);
+    const duration = player.seekRange().end
```

Comment:
@joeyparrish should i be using this to determine the time??
also is there any other way to check for this, like some function that tells the number of periods in the manifest?

Reply:
There is no way to check the number of periods in Shaka v3.  In Shaka v2, we kept periods separate, but now we combine them in the player when we parse DASH.